interfaces/builtin/opengl_test.go

```diff
@@ -87,6 +90,17 @@ func (s *OpenglInterfaceSuite) TestAppArmorSpec(c *C) {
 	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, `/dev/dma_heap/system rw,`)
 	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, `/dev/galcore rw,`)
 	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, `/usr/share/libdrm/amdgpu.ids r,`)
+	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, `/usr/share/nvidia/ r,`)
+	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, `/usr/share/nvidia/** r,`)
+
+	updateNS := spec.UpdateNS()
+
+	// This all gets added as one giant snippet so just testing for the comment fails
+	c.Check(updateNS, testutil.Contains, `  # Read-only access to Nvidia driver profiles in /usr/share/nvidia
+	mount options=(bind) /var/lib/snapd/hostfs/usr/share/nvidia/ -> /usr/share/nvidia/,
+	remount options=(bind, ro) /usr/share/nvidia/,
+	umount /usr/share/nvidia/,
```

Comment:
This part is failing in CI, since the UpdateNS rules are only added if `/usr/share/nvidia` exists. There's two ways you could fix this:

1. mock the path like you're doing in `TestMountSpec`
2. always add the UpdateNS AppArmor rules: having them shouldn't hurt if we don't generate the mount.